### DISCUSSION TITLE
Average wait time for client to change channel when sending probe requests

### DISCUSSION THREAD

**Discussion by vertexgamer** [ZerBea/hcxdumptool#422] [Q&A] [answered]
Sup, since i only have one adapter and can't reliably monitor multiple channel at once, does anyone has information about the average wait time of clients before they change channel when sending probe requests? I read online that probe requests are sent to all channels sequentially.

**Comment by ZerBea** [comment, marked as answer]
It's hard to answer that question, because every client is doing this in a different way:
https://developer.android.com/develop/connectivity/wifi/wifi-scan
https://stackoverflow.com/questions/29382566/how-often-does-android-system-scan-wifi
https://android.stackexchange.com/questions/37621/where-can-i-find-settings-for-wifi-internal-scan-period

BTW:
All my test CLIENTs do a periodic scan while the are connected to the test target AP.
If hcxdumptool offers better conditions they try to reassociate to hcxdumptool.

Due to heavy interferences it is not a good idea to run more than one adapter.
If you're hunting for a special CLIENT I suggest to increase the stay time on the channel. I'm sure, the CLIENT will find hcxdumptool.
